platform/packages/platform/src/contract.rs

```diff
@@ -136,12 +136,13 @@ pub mod testing {
     pub fn valid_contract_handler(_query: &WasmQuery) -> QuerierResult {
```

Comment:
The parameter is not used at all in any case, so we can just discard as a whole it instead.
```suggestion
    pub fn valid_contract_handler(_: &WasmQuery) -> QuerierResult {
```